File: clean-master-web-app/src/c/virus_scanner.c
#include <stdio.h>
#include <string.h>

void scan_for_virus(const char *filename) {
    FILE *file = fopen(filename, "r");
    char buffer[1024];
    const char *virus_signature = "Trojan1234";

    if (file == NULL) {
        printf("Dosya açılmadı\n");
        return;
    }

    while (fgets(buffer, sizeof(buffer), file)) {
        if (strstr(buffer, virus_signature)) {
            printf("Virüs tespit edildi: %s\n", virus_signature);
        }
    }
    fclose(file);
}

int main() {
    scan_for_virus("test_file.txt");
    return 0;
}

File: clean-master-web-app/src/php/virus_scanner.php
<?php
function scan_for_virus($filename) {
    $virus_signature = "Trojan1234";
    $file = fopen($filename, "r");

    if ($file) {
        while (($line = fgets($file)) !== false) {
            if (strpos($line, $virus_signature) !== false) {
                echo "Virüs tespit edildi: " . $virus_signature . "\n";
            }
        }
        fclose($file);
    } else {
        echo "Dosya açılamadı.\n";
    }
}

scan_for_virus("test_file.txt");
?>

